client/schemas/vote.ts

```diff
@@ -66,6 +66,20 @@ export const voteSchema = {
       enum: Object.values(OverallVote),
       title: 'Overall Vote',
     },
-    comment: { type: 'string', title: 'Comment' },
+    comment: {
+      type: 'string',
+      title: 'Comment',
+      default: 'Fit: \nMarket: \nProduct: \nTeam: \nOverall:',
+    },
+  },
+};
+
+export const voteUISchema = {
```

Comment:
Let's leave a comment here on what this is specifying

Reply:
Done.